src/hyperlight_host/src/hypervisor/hyperv_linux.rs

```diff
@@ -438,6 +476,13 @@ impl Hypervisor for HypervLinuxDriver {
                         None => HyperlightExit::Mmio(gpa),
                     }
                 }
+                #[cfg(gdb)]
```

Comment:
Might be worth commenting here that the only case where we are expecting an intercept exit is the case we are debugging as we are only enabling exception exits for debug exceptions?

Reply:
yes, I'll add a comment